<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDecryptionAndVerifyEditor</class>
 <widget class="QDialog" name="FileDecryptionAndVerifyEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DÃ©chiffrer</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>224</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="DialogButtonLayout3">
    <item>
     <spacer name="horizontalSpacer3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="acceptButton2">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton2">
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="destinationFileLabel2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>108</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier destination</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="destinationFileEdit2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>237</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseDestinationFileButton2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcourir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
